dev-cpp/yaml-cpp/yaml-cpp-0.7.0-r2.ebuild

```diff
@@ -20,15 +20,10 @@ DEPEND="test? ( dev-cpp/gtest[${MULTILIB_USEDEP}] )"
 
 PATCHES=(
 	"${FILESDIR}/${P}-gtest.patch"
-	"${FILESDIR}/${P}-pkg-config.patch"
+	"${FILESDIR}/${P}-cmake-paths.patch"
+	"${FILESDIR}/${P}-install-paths.patch"
 )
 
-src_prepare() {
-	rm -r test/gtest-* || die
```

Comment:
Why removing this?

Reply:
See https://github.com/gentoo/gentoo/pull/27302#discussion_r973583905